trident/line_database.py

```diff
@@ -143,6 +179,30 @@ class LineDatabase:
     >>> print(lines)
 
     """
+    @classmethod
+    def from_lt(cls, line_list):
+        """Initialize from a linetools LineList object.
+
+        **Parameters**
+
+        :line_list: linetools.lists.linelist.LineList
+
+        **Example**
+
+        >>> # Create a LineDatabase from a linetools LineList
+        >>> from linetools.lists.linelist import LineList
+        >>> ldb = LineDatabase.from_lt(LineList('ISM'))
+
+        """
+        # Start with an empty database
+        ldb = cls()
+        # Add lines manually
+        for wrest in line_list.wrest:
```

Comment:
I feel this way of iterating through LineList is a little bit hacky. Is there a standard way of doing this in linetools?

Reply:
Indeed this method is problematic. I tried
```python
LineDatabase.from_lt(LineList('EUV'))
```
which would error
```
ValueError: LineList: Multiple lines in the list with your input.  Give a more unique input or change the tol.
```
So there's a potential issue that `wrest` is not unique in the line list. Although for practical use, this might rarely happen.